### PR TITLE
Mohanraj

### PR DESCRIPTION
**Pull request by Mohanraj3495**
please review my changes and commit with dev branch sir

### CODE REVIEW

File: Caller/ViewController.swift [hunk 1]
```diff
@@ -9,18 +9,29 @@ import Cocoa
...
+    let value2 = 20
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
         // Do any additional setup after loading the view.
     }
+    
+    deinit{
+        print("Deinit viewcontroller")
```
Comment (Saran-ghub) [inline Caller/ViewController.swift]: Don't push deinit.

Comment (Mohanraj3495) [reply Caller/ViewController.swift]: okay sir

Comment (Mohanraj3495) [reply Caller/ViewController.swift]: removed deinit kindly check again @Saran-ghub

**Review by Saran-ghub — APPROVED** [review]
Looks good!

**Pull request merged**